content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -780,43 +777,43 @@ These are some requirements and semantics of the API:
 
 ### Job tracking with finalizers
 
-{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
 {{< note >}}
-In order to use this behavior, you must enable the `JobTrackingWithFinalizers`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
-and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/).
-
-When enabled, the control plane tracks new Jobs using the behavior described
-below. Jobs created before the feature was enabled are unaffected. As a user,
-the only difference you would see is that the control plane tracking of Job
-completion is more accurate.
+The control plane doesn't track Jobs using finalizers, if the Jobs were created
+when the feature gate `JobTrackingWithFinalizers` was disabled, even after
+upgrading the control plane to 1.26.
 {{< /note >}}
 
-When this feature isn't enabled, the Job {{< glossary_tooltip term_id="controller" >}}
+In Kubernetes 1.25 or older, where the feature gate `JobTrackingWithFinalizers`
+doesn't exist or is disabled, the Job {{< glossary_tooltip term_id="controller" >}}
 relies on counting the Pods that exist in the cluster to track the Job status,
-that is, to keep the counters for `succeeded` and `failed` Pods.
-However, Pods can be removed for a number of reasons, including:
+that is, to keep the counters for `succeeded` and `failed` Pods. The control
+plane can lose track of the progress of the Job if Pods are deleted from the
+cluster.
+Pods can be removed for a number of reasons, including:
 - The garbage collector that removes orphan Pods when a Node goes down.
 - The garbage collector that removes finished Pods (in `Succeeded` or `Failed`
   phase) after a threshold.
 - Human intervention to delete Pods belonging to a Job.
 - An external controller (not provided as part of Kubernetes) that removes or
   replaces Pods.
 
-If you enable the `JobTrackingWithFinalizers` feature for your cluster, the
-control plane keeps track of the Pods that belong to any Job and notices if any
-such Pod is removed from the API server. To do that, the Job controller creates Pods with
-the finalizer `batch.kubernetes.io/job-tracking`. The controller removes the
-finalizer only after the Pod has been accounted for in the Job status, allowing
+In Kubernetes 1.26 or newer (1.23 to 1.25, if you enable the
+`JobTrackingWithFinalizers` feature gate), the control plane keeps track of the
+Pods that belong to any Job and notices if any such Pod is removed from the API
+server. To do that, the Job controller creates Pods with the finalizer
+`batch.kubernetes.io/job-tracking`. The controller removes the finalizer only
+after the Pod has been accounted for in the Job status, allowing
```

Comment:
We usually document just for the latest version, and add a sentence at the start of the page that if
you're not running Kubernetes `{{< skew currentVersion >}}` you should then go and read the docs for the version you _are_ running.

Reply:
Sounds good. I refactored the entire section.